packages/app-project/stores/User/UserPersonalization/UserPersonalization.spec.js

```diff
@@ -45,12 +56,14 @@ describe('Stores > UserPersonalization', function () {
 
     rootStore = initStore(true, { project })
     sinon.spy(rootStore.client.panoptes, 'get')
+    sinon.stub(statsClient, 'fetchDailyStats').callsFake(() => Promise.resolve({ data: MOCK_DAILY_COUNTS }))
```

Comment:
Maybe add the expected parameters to your mock here.

eg.
```suggestion
    sinon.stub(statsClient, 'fetchDailyStats').callsFake(({ projectId, userId }) => (projectId === '2' && userId === '123') ?
      Promise.resolve({ data: MOCK_DAILY_COUNTS })) :
      Promise.reject(new Error(`Unable to fetch stats for project ${projectId} and user ${userId}`))
```